packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -59,22 +59,35 @@ The user is emailed a one-time activation token if activated without a password.
 | Security Q & A   | Password   | Activate Query Parameter   | User Status     | Login Credential         | Welcome Screen   |
 | :--------------: | :--------: | :------------------------: | :-------------: | :----------------------: | :--------------: |
 |                  |            | FALSE                      | `STAGED`        |                          |                  |
-|                  |            | TRUE                       | `PROVISIONED`   | One-Time Token (Email)   | X                |
+|                  |            | TRUE                       | `PROVISIONED` or `ACTIVE`   | One-Time Token (Email) or Email  | X                |
 | X                |            | FALSE                      | `STAGED`        |                          |                  |
-| X                |            | TRUE                       | `PROVISIONED`   | One-Time Token (Email)   | X                |
+| X                |            | TRUE                       | `PROVISIONED` or `ACTIVE`   | One-Time Token (Email) or Email  | X                |
 |                  | X          | FALSE                      | `STAGED`        |                          |                  |
 |                  | X          | TRUE                       | `ACTIVE`        | Password                 | X                |
 | X                | X          | FALSE                      | `STAGED`        |                          |                  |
 | X                | X          | TRUE                       | `ACTIVE`        | Password                 |                  |
 
 Creating users with a `FEDERATION` or `SOCIAL` provider sets the user status to either `ACTIVE` or `STAGED` based on the `activate` query parameter since these two providers don't support a `password` or `recovery_question` credential.
 
+#### Create User with Optional Password enabled
+
+<ApiLifeCycle access="ea" />
+
+When Optional Password is enabled, user status following creation can be affected by enrollment policy (see [About MFA enrollment policies and rules](/Content/Topics/identity-engine/policies/about-mfa-enrollment-policies.htm)).
```

Comment:
I don't know how to input the link in this case, because I don't think its part of the API reference repo.
The full link is: https://help.okta.com/oie/en-us/Content/Topics/identity-engine/policies/create-mfa-policy.htm.
Should I use that?